.gitignore

```diff
@@ -0,0 +1 @@
+scripts/technique-files
```

Comment:
This file is now downloaded by Makefile from a different repo, so shouldn't be tracked here